region/brinstar/red/Red Tower.json

```diff
@@ -2285,17 +2285,27 @@
       "link": [7, 6],
       "name": "SpringBall Jump",
```

Comment:
This strat has a version that can be done without wall jump or HiJump: https://videos.maprando.com/video/4228